scripts/build.sh

```diff
@@ -42,7 +42,7 @@ fi
 if [[ $# -eq 1 ]]; then
     binary_path=$1
 elif [[ $# -eq 0 ]]; then
-    binary_path="build/awm-relayer"
+    binary_path=$relayer_path
```

Comment:
Did you mean to change this?

Reply:
I didn't mean to commit this as part of the PR, but now that it snuck in. I was curious why do we set the variable in `constants.sh` file and then never use it.